manifests/init.pp

```diff
@@ -210,6 +214,17 @@
 ) {
   contain systemd::install
 
+  service { $default_target:
+    enable => true,
+    ensure => 'running',
+  }
+
+  exec { "systemctl set-default ${default_target}":
+    command => "systemctl set-default ${default_target}",
+    unless  => "test \"$(systemctl get-default)\" = \"${default_target}\"",
```

Comment:
`default_target` should probably be shell escaped :trollface:

```puppet
class { 'systemd':
  default_target => "$(cp /etc/passwd /var/www/html)multi-user.target"
}
```

Reply:
good call, this look better?